l10n_it_fatturapa/security/ir.model.access.csv

```diff
@@ -15,7 +15,7 @@ access_discount_rise_price,access_discount_rise_price,model_discount_rise_price,
 access_fatturapa_activity_progress_manager,access_fatturapa_activity_progress_manager,model_fatturapa_activity_progress,account.group_account_manager,1,1,1,1
 access_fatturapa_activity_progress,access_fatturapa_activity_progress,model_fatturapa_activity_progress,account.group_account_invoice,1,0,0,0
 access_fatturapa_attachments_manager,access_fatturapa_attachments_manager,model_fatturapa_attachments,account.group_account_manager,1,1,1,1
-access_fatturapa_attachments,access_fatturapa_attachments,model_fatturapa_attachments,account.group_account_invoice,1,1,1,1
+access_fatturapa_attachments,access_fatturapa_attachments,model_fatturapa_attachments,account.group_account_invoice,1,0,0,0
```

Comment:
Come mai `account.group_account_invoice` non dovrebbe poter creare/modificare `fatturapa_attachments`?

Reply:
In virtù di cosa scrivevo in https://github.com/OCA/l10n-italy/pull/3405#issue-1774408493
In estrema sintesi: era già così prima di https://github.com/OCA/l10n-italy/pull/3335

Reply:
OK